_posts/2018-04-09-Rails-5-2-0-final.markdown

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: 'Rails 5.2.0 FINAL: Active Storage, Redis Cache Store, HTTP/2 Early Hints, CSP, Credentials'
+categories: releases
+author: dhh
+published: true
+date: 2018-04-09 12:00:00 -09:00
+---
+Nearly 14 years since [the first public version of Rails](http://blade.nagaokaut.ac.jp/cgi-bin/scat.rb/ruby/ruby-talk/107370), it's our pleasure to release yet another major upgrade to the framework in the form of 5.2.0 final. We've been diligently polishing Active Storage and the other big new components for stable release, and it's great to see so many applications already running the release candidates in production. [Basecamp](https://basecamp.com) and [Shopify](https://shopify.com) have both been running Rails 5.2.0 for quite a while.
+
+This release comes just in time for [RailsConf](https://railsconf.com/), which features sessions on the new [encrypted credentials](https://railsconf.com/program/sessions#session-627), [a code review of Active Storage](https://railsconf.com/program/sessions#session-562), [advice on how to upgrade to a new Rails version](https://railsconf.com/program/sessions#session-588), and a [lot](https://railsconf.com/program/sessions#session-552) [of](https://railsconf.com/program/sessions#session-568) [Webpack](https://railsconf.com/program/sessions#session-549) talks.
+
+You can read in even more detail about everything that's [new in Rails 5.2 in the newly finished release notes](http://edgeguides.rubyonrails.org/5_2_release_notes.html).
+
+Note that [rails/master](https://github.com/rails/rails) development is now targeting Rails 6.0.
+
+Many thanks to Rails core, Rails contributors, and everyone else who've helped with code, documentation, bug reports, and whatever else to get Rails 5.2.0 out the door. It's amazing to have [over 500 code contributors](http://contributors.rubyonrails.org/edge/contributors) with fingerprints on this release.
```

Comment:
The link to the edge contributors now points to 6.0. I'll fix it as soon the 5.2.0 link is generated.